Add Functional test example in README (#71)
ddev/ddev-selenium-standalone-chrome: README.md

## Overview

> Makes it super easy to run Functional, FunctionalJavascript, and Nightwatch tests in DDEV. You can follow along in the browser via noVNC.

Reviewer: ```suggestion
Makes it easy to run Functional, FunctionalJavascript, and Nightwatch tests in DDEV. 
```

ddev add-on get ddev/ddev-selenium-standalone-chrome
ddev restart
```

> Functional and FunctionalJavascript requires `drupal/core-dev` Composer package or equivalent:

Reviewer: ```suggestion
Functional and FunctionalJavascript tests require the `drupal/core-dev` Composer package or equivalent:
```

## Usage

- Your project is now ready to run [Functional](https://mglaman.dev/blog/do-you-need-functional-test), FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://gitlab.com/weitzman/drupal-test-traits) (DTT). All these types are tested in this repo. Some examples to try:
  - Functional and FunctionalJavascript:
    - Ensure you have the `drupal/core-dev` Composer package or equivalent.
    - `ddev exec -d /var/www/html/web "../vendor/bin/phpunit -c ./core/phpunit.xml.dist ./core/modules/migrate/tests/src/Functional/process/DownloadFunctionalTest.php"`
    - `ddev exec -d /var/www/html/web "../vendor/bin/phpunit -c ./core/phpunit.xml.dist ./core/modules/system/tests/src/FunctionalJavascript/FrameworkTest.php"`
  - Nightwatch

>     - `ddev exec -d /var/www/html/web/core yarn install` (do this once)

Reviewer: Lets break this into three bullets instead of grouping Functional and FunctionalJavascript

## Usage

> - Your project is now ready to run [Functional](https://mglaman.dev/blog/do-you-need-functional-test), FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://git.drupalcode.org/project/dtt) (DTT). All these types are tested in this repo. Some examples to try:

Reviewer: We dont yet test Functional in this repo.

Author: Great catch, I have tried to clarify this.

> - Your project is now ready to run [Functional](https://mglaman.dev/blog/do-you-need-functional-test), FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://git.drupalcode.org/project/dtt) (DTT). FunctionalJavascript and Nightwatch tests types are tested in this repo. Some examples to try:

Reviewer: ```suggestion
- Your project is now ready to run [Functional](https://mglaman.dev/blog/do-you-need-functional-test), FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://git.drupalcode.org/project/dtt) (DTT). Some examples to try:
```
